Forge.TreeWalker/Forge.TreeWalker.csproj

```diff
@@ -58,7 +58,7 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
     <PackageReference Include="System.Collections.Immutable" Version="1.2.0" />
-    <PackageReference Include="System.Reflection.Metadata" Version="1.2.0" >
+    <PackageReference Include="System.Reflection.Metadata" Version="1.2.0">
```

Comment:
>"> [](start = 73, length = 2)

nit: add space back.

Reply:
The visual studio 2019 auto removed this space. After double check, looks like the space actually should be removed because this is a complete XML element (<A></A>), not a compact one (<A />)

---
In reply to: [504219758](https://github.com/microsoft/Forge/pull/32#discussion_r504219758) [](ancestors = 504219758)